shader/lanczos3.frag

```diff
@@ -2,6 +2,7 @@
 // Copyright 2020 Lilian Gimenez (Sentmoraap).
 // MIT license.
 
+precision highp float;
```

Comment:
One https://github.com/mkxp-z/mkxp-z/pull/119 is merged, I think we should be able to remove this workaround.